boards/arm/efm32pg_stk3402a/efm32pg_stk3402a-map.yml

```diff
@@ -0,0 +1,10 @@
+- connected: true
```

Comment:
What's the purpose of this file?

Reply:
Our backend pipeline needs fixed values for what is being installed in our board farm, and our developers need a place to impose their value for testing, which should never be committed.

Reply:
This should be kept in another repository than the main Zephyr repository.

Reply:
I've moved forward with splitting this PR into multiple smaller PRs. The 1st of which is here: [boards: arm: Update efm32pg_stk3402a to run twister tests that pass](https://github.com/zephyrproject-rtos/zephyr/pull/58043), and it does still contain this file, but I hear you and I'll figure out a better way.